lib/database_cleaner/mongo2/truncation_mixin.rb

```diff
@@ -0,0 +1,37 @@
+module DatabaseCleaner
+  module Mongo2
+    module TruncationMixin
+
+      def clean
+        if @only
+          collections.each { |c| database[c].find.delete_many if @only.include?(c) }
+        else
+          collections.each { |c| database[c].find.delete_many unless @tables_to_exclude.include?(c) }
+        end
+        true
+      end
+
+      private
+
+      def database
+        if not(@db.nil? or @db == :default)
```

Comment:
@jprincipe Could you make this condition without the `not`? I think it'd make it easier to understand.